### PR TITLE
anagram: Change property to be more descriptive

### PR DESCRIPTION
**Pull request by Stargator**
I find `anagrams` to be a generic name for a method or function. I think the property should be description of either the action or the result.

Since the property is about returning a list of anagrams found of a given word, I think `findAnagrams` would be alright.

### CODE REVIEW

File: exercises/anagram/canonical-data.json [hunk 1]
```diff
@@ -1,10 +1,10 @@
 {
   "exercise": "anagram",
-  "version": "1.2.1",
+  "version": "1.3.1",
```
Comment (rpottsoh) [inline exercises/anagram/canonical-data.json]: Should be 1.3.0 I think.

**Comment by rpottsoh** [comment]
`findAnagrams` is fine to me. :+1:

**Comment by macta** [comment]
Having these kinds of names will make it much easier for many tracks to auto generate test routes. Thanks

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by ErikSchierboom** [comment]
I'm with @petertseng on this, where I also slightly prefer `anagrams` over `findAnagrams`, although I find it hard to articulate why. Don't let that deter you from merging this, as my preference is not strong.

**Comment by petertseng** [comment]
I was about to say "but the anagrams exist independent of any action; no verb is required for the anagrams to exist". However, I then refreshed my memory of the exercise, and the implementation needs to pick out the anagrams from a list of candidates, so that can be argued that that needs an action.

So, my preference for `anagrams` over `findAnagrams` is now downgraded to 1/10

**Pull request merged**